tests/resources/app_structure/suite.xml

```diff
@@ -6,6 +6,12 @@
     </resource>
   </xform>
 
+  <user-restore username="test" password="123">
+    <resource id="foo" version="1" descriptor="demo user restore">
+      <location authority="local">./user_restore.xml</location>
```

Comment:
That's because it already exists prior to this PR. I'm actually not sure why this block was even added to the suite.xml here though, since it's not being used anywhere for tests